server/protocols/jmap/pom.xml

```diff
@@ -45,6 +45,10 @@
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
so now jmap depends on mailbox implementation details ... not a good news.

Reply:
I don't know how can deal with this situation when we want EventListener only on JMAP :(

Reply:
MailboxListener is in mailbox-api as expected. Which API is only in store ?

Reply:
ok, fixed that, it turns out we needed `MailboxACL RightsManager.listRights()`